docs/examples/incremental_time_full_walkthrough.md

```diff
@@ -1,4 +1,4 @@
-# Incremental by Time Range: Full Walkthrough
+# Incremental by Time Range
```

Comment:
```suggestion
# Incremental by Time Range: Full Walkthrough
```

Adding this back in as this is the first example post and don't want to set the precedent that this will only include full walkthroughs.

Reply:
Nevermind, just saw you updated the nav bar